packages/core/src/modules/indy/services/IndyHolderService.ts

```diff
@@ -205,6 +206,24 @@ export class IndyHolderService {
     }
   }
 
+  /**
+   * Delete a credential stored in the wallet by id.
+   *
+   * @param credentialId the id (referent) of the credential
+   *
+   */
+  public async deleteCredential(credentialId: Indy.CredentialId): Promise<void> {
+    try {
+      return await this.indy.deleteWalletRecord(this.wallet.handle, CredentialRecord.type, credentialId)
```

Comment:
You need to call `proverDeleteCredential`

https://github.com/hyperledger/indy-sdk/blob/master/wrappers/nodejs/src/index.js#L188-L192

Reply:
`indy.proverDeleteCredential` is not exposed.. ?